components/proxy/plugins/analytics/analytics.go

```diff
@@ -24,6 +24,8 @@ import (
 
 const (
 	moduleName = "gitpod.analytics"
+	// static key for untrusted segment requests
+	dummyUntrustedSegmentKey = "ge5jLIQ+ZR9My3guykQu6XH6hDI"
```

Comment:
Unless it needs to pass a particular integrity check, I'd even make this a plain "untrusted-dummy-key" just to make it easier when searching/encountering it